addons/barcodes_gs1_nomenclature/data/barcodes_gs1_rules.xml

```diff
@@ -334,5 +334,873 @@
+            <field name="associated_uom_id" ref="uom.product_uom_unit"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
+            <field name="active" eval="False"/>
+        </record>
+
+        <record id="barcode_rule_gs1_from_340y_to_349y" model="barcode.rule">
+            <field name="name">Unused GS1 AI measure rules</field>
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3400</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(34[0-9][0-5])(\d{6})</field>
+            <field name="associated_uom_id" ref="uom.product_uom_unit"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
+        </record>
+
+        <record id="barcode_rule_gs1_350y_and_from_352y_to_356y" model="barcode.rule">
+            <field name="name">Unused GS1 AI measure rules</field>
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3500</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(35[02-6][0-5])(\d{6})</field>
+            <field name="associated_uom_id" ref="uom.product_uom_unit"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
+            <field name="active" eval="False"/>
+        </record>
+
+        <record id="barcode_rule_gs1_362y_363y_and_from_366y_to_369y" model="barcode.rule">
+            <field name="name">Unused GS1 AI measure rules</field>
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3620</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(36[236-9][0-5])(\d{6})</field>
+            <field name="associated_uom_id" ref="uom.product_uom_unit"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
+            <field name="active" eval="False"/>
+        </record>
+
+        <record id="barcode_rule_gs1_390y_392y" model="barcode.rule">
+            <field name="name">Unused GS1 AI applicable amount payable rules</field>
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3900</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(39[02][0-9])(\d{0,15})</field>
+            <field name="associated_uom_id" ref="uom.product_uom_unit"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
```

Comment:
I think some of these are leftover from copy paste:

```suggestion
            <field name="type">skip</field>
            <field name="gs1_content_type">identifier</field>
```

Reply:
It was wanted (see [previous comment](https://github.com/odoo/odoo/pull/158538#discussion_r1570344520)) but surely wrong :sweat_smile: 
IMO `identifier` is not a better option and sadly we have no good option for price right now 🫤